### ISSUE TITLE
Getting the default theme for Node-RED when accessing instances running on device agent 2.0

### ISSUE THREAD

**Issue by robmarcer** [FlowFuse/device-agent#229]
### Current Behavior

https://d.pr/i/Sdld9v

### Expected Behavior

I think I should see the FF NR theme.

### Steps To Reproduce

Installed version 2.0.0 of the device agent in docker.
Using 3.1.3 Node-RED

I noted this in the NR logs:
```
[AGENT] 1/31/2024 1:54:43 PM [info] Could not write theme files. Theme not found: '/usr/local/lib/node_modules/@flowfuse/nr-theme'
```

### Environment

- FlowFuse version: v2.0.2-6b0e36b-202401291455.0
- Node.js version: v18.19.0
- npm version:
- Platform/OS: Debian Bullseye / Docker / FF Device Agent 2.0.0
- Browser:

**Comment by Steve-Mcl** [comment]
@robmarcer this looks to be a duplicate of #224 and is fixed in #235

**Comment by Steve-Mcl** [comment]
closing as fixed by https://github.com/FlowFuse/device-agent/pull/235
